### DISCUSSION TITLE
How can we maintain at least a release cycle of once a week

### DISCUSSION THREAD

**Discussion by prjctimg** [prjctimg/huetiful#86] [Q&A] [answered]
I've noticed that the project doesn't release often enough to stay afloat in terms of traffic on both GitHub and NPM. To improve our outreach, how can we maintain a development pace that allows us to release patches or updates to the library at least once a week. This observation was inspired by open-sauced which mantains a very active release cycle. 

Ideas and suggestions for workflow management are greatly welcome! 

Let's make this project active and thriving! 🚀

**Comment by takanome-dev** [comment, marked as answer]
Great observation!! 

I'm not very familiar with the release cycle there but I can ask the maintainers if/how we can use it and personalize it for our needs.

**Reply by prjctimg** [reply]
> Great observation!! 
> 
> I'm not very familiar with the release cycle there but I can ask the maintainers if/how we can use it and personalize it for our needs.

And while we're still at it 😂, the publishing, building and testing should be handled by a workflow file whenever we create a new git tag. 

Automating the development process as much as possible will give us time to explore other ideas!

**Reply by takanome-dev** [reply]
[Vortex](https://github.com/0-vortex), who created the release cycle, is going to implement it in one of his Nextjs project. Just waiting for him to open source then I will copy it here

**Reply by prjctimg** [reply]
> [Vortex](https://github.com/0-vortex), who created the release cycle, is going to implement it in one of his Nextjs project. Just waiting for him to open source then I will copy it here

Awesome! Pardon the late response. Please keep us updated! 

Released v1. 7.5 just now addressing a lot of changes!
